addons/l10n_in_sale/wizard/sale_make_invoice_advance.py

```diff
@@ -7,8 +7,8 @@
 class SaleAdvancePaymentInv(models.TransientModel):
     _inherit = "sale.advance.payment.inv"
 
-    def _create_invoice(self, order, so_line, amount):
-        invoice = super()._create_invoice(order, so_line, amount)
+    def _prepare_invoice_values(self, order, name, amount, so_line):
+        res = super()._create_invoice(order, so_line, amount)
```

Comment:
bad super call, you're generating an infinite recursion since the call to `_create_invoice` on `super` will call `_prepare_invoice_values` which will call `_create_invoice` etc...
```py
def _prepare_invoice_values(self, order, name, amount, so_line):
    res = super()._prepare_invoice_values(order, so_line, amount)
```
if you want me to merge your code, please test it; the testing flow takes literally 2min to set up:
- create a company in India
- install the Indian chart of account and enable it for this company
- create a quote and set a journal on the quote (the field is added by `l10n_in`), confirm it
- generate a downpaymnt and try to invoice it: the downpayment invoice must have the journal set on the quote

As long as this flow doesn't work properly (it currently crashes when creating the downpayment invoice), I will not merge this

Reply:
@dbo-odoo hey I'm very sorry for that man, I always missed one of the steps which means it was always taking the normal main flow which obviously just works fine. It never got through `l10n_in_sale` which I actually didn't notice /dumb.
I've corrected the super call and it seems to work fine now on my local instance. I'll do a second testrun after runbot is built and green too. 
Sorry ;(